.github/workflows/job_build_windows.yml

```diff
@@ -176,6 +180,54 @@ jobs:
           cmake --install . --config ${{ env.CMAKE_BUILD_TYPE }} --prefix ${{ env.INSTALL_TEST_DIR }} --component tests
         working-directory: ${{ env.BUILD_DIR }}
 
+      # Setup additional Python versions for wheels building
+      - name: Setup Python 3.9
```

Comment:
Can we use python matrix?

Reply:
No, as the matrix is only available for the jobs and not for steps.